src/library.test.js

```diff
@@ -32,13 +31,16 @@ describe('deployment status', () => {
     }
   });
 
-  test.each(ALLOWED_STATUSES)(`valid deployment status of %s does not throw exception`, status => {
-    process.env[inputKey('deployment-status')] = status;
-    try {
-      const testConext = setup();
-      expect(testConext.deployment_status).toBe(status);
-    } catch (error) {
-      expect(error).toBeUndefined();
+  test.each(Object.values(ALLOWED_STATUSES))(
+    `valid deployment status of %s does not throw exception`,
+    status => {
+      process.env[inputKey('deployment-status')] = status;
+      try {
+        const testConext = setup();
```

Comment:
This is a nitpick but it looks like there is a typo here. `testConext` => `testContext`.

Reply:
👍🏽